int32 id
int32 cur_x
int32 cur_y
int32 cur_theta
int32 goal_x
int32 goal_y
int32 goal_theta
---
string path
string path_y
int32 path_node_num
int32 agent_id